application/app/Http/Controllers/TaskController.php

```diff
@@ -145,16 +147,18 @@ public function show(Project $project, Task $task)
      * @param  \App\Models\Task  $task
      * @return \Illuminate\Http\Response
      */
-    public function edit(Project $project, Task $task)
+    public function edit(Project $project, Task $task )
```

Comment:
```suggestion
    public function edit(Project $project, Task $task)
```
このファイル `TaskController.php` 全体をPHP Intelephenseで自動整形しましょう。半角スペースを整形したいので。